src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1266,4 +1266,22 @@
   <data name="DataSetReadXmlAutogeneratedTitle" xml:space="preserve">
     <value>Ensure autogenerated class containing DataSet.ReadXml() is not used with untrusted data</value>
   </data>
+  <data name="DoNotDisableTokenValidationChecks" xml:space="preserve">
+    <value>Do Not Disable Token Validation Checks</value>
```

Comment:
I think this should be sentence casing, ie `Do not disable token validation checks`

Reply:
Thanks! I was patterning off of "DoNotDisableCertificateValidation::Do Not Disable Certificate Validation" but looking again that seems like an anomaly. Will update this and the other.